Traceback (most recent call last):
  File "/Users/kvarada/opt/miniconda3/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/kvarada/opt/miniconda3/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/kvarada/opt/miniconda3/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/kvarada/opt/miniconda3/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/kvarada/opt/miniconda3/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/Users/kvarada/opt/miniconda3/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/kvarada/opt/miniconda3/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/kvarada/opt/miniconda3/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import wikipedia
from nltk.tokenize import sent_tokenize, word_tokenize

corpus = []

queries = ["Machu Picchu", "human stature", "Everest"]

for i in range(len(queries)):
    sents = sent_tokenize(wikipedia.page(queries[i]).content)
    corpus.extend(sents)
print("Number of sentences in the corpus: ", len(sents))
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mPageError[0m                                 Traceback (most recent call last)
[0;32m/var/folders/ky/533nd9l512l5cmmk_kzmzj9m0000gp/T/ipykernel_97187/4015128425.py[0m in [0;36m<module>[0;34m[0m
[1;32m      7[0m [0;34m[0m[0m
[1;32m      8[0m [0;32mfor[0m [0mi[0m [0;32min[0m [0mrange[0m[0;34m([0m[0mlen[0m[0;34m([0m[0mqueries[0m[0;34m)[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 9[0;31m     [0msents[0m [0;34m=[0m [0msent_tokenize[0m[0;34m([0m[0mwikipedia[0m[0;34m.[0m[0mpage[0m[0;34m([0m[0mqueries[0m[0;34m[[0m[0mi[0m[0;34m][0m[0;34m)[0m[0;34m.[0m[0mcontent[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     10[0m     [0mcorpus[0m[0;34m.[0m[0mextend[0m[0;34m([0m[0msents[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m     11[0m [0mprint[0m[0;34m([0m[0;34m"Number of sentences in the corpus: "[0m[0;34m,[0m [0mlen[0m[0;34m([0m[0msents[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/opt/miniconda3/envs/cpsc330/lib/python3.9/site-packages/wikipedia/wikipedia.py[0m in [0;36mpage[0;34m(title, pageid, auto_suggest, redirect, preload)[0m
[1;32m    274[0m         [0;31m# if there is no suggestion or search results, the page doesn't exist[0m[0;34m[0m[0;34m[0m[0m
[1;32m    275[0m         [0;32mraise[0m [0mPageError[0m[0;34m([0m[0mtitle[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 276[0;31m     [0;32mreturn[0m [0mWikipediaPage[0m[0;34m([0m[0mtitle[0m[0;34m,[0m [0mredirect[0m[0;34m=[0m[0mredirect[0m[0;34m,[0m [0mpreload[0m[0;34m=[0m[0mpreload[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    277[0m   [0;32melif[0m [0mpageid[0m [0;32mis[0m [0;32mnot[0m [0;32mNone[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    278[0m     [0;32mreturn[0m [0mWikipediaPage[0m[0;34m([0m[0mpageid[0m[0;34m=[0m[0mpageid[0m[0;34m,[0m [0mpreload[0m[0;34m=[0m[0mpreload[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/opt/miniconda3/envs/cpsc330/lib/python3.9/site-packages/wikipedia/wikipedia.py[0m in [0;36m__init__[0;34m(self, title, pageid, redirect, preload, original_title)[0m
[1;32m    297[0m       [0;32mraise[0m [0mValueError[0m[0;34m([0m[0;34m"Either a title or a pageid must be specified"[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    298[0m [0;34m[0m[0m
[0;32m--> 299[0;31m     [0mself[0m[0;34m.[0m[0m__load[0m[0;34m([0m[0mredirect[0m[0;34m=[0m[0mredirect[0m[0;34m,[0m [0mpreload[0m[0;34m=[0m[0mpreload[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    300[0m [0;34m[0m[0m
[1;32m    301[0m     [0;32mif[0m [0mpreload[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/opt/miniconda3/envs/cpsc330/lib/python3.9/site-packages/wikipedia/wikipedia.py[0m in [0;36m__load[0;34m(self, redirect, preload)[0m
[1;32m    343[0m     [0;32mif[0m [0;34m'missing'[0m [0;32min[0m [0mpage[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    344[0m       [0;32mif[0m [0mhasattr[0m[0;34m([0m[0mself[0m[0;34m,[0m [0;34m'title'[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 345[0;31m         [0;32mraise[0m [0mPageError[0m[0;34m([0m[0mself[0m[0;34m.[0m[0mtitle[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    346[0m       [0;32melse[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    347[0m         [0;32mraise[0m [0mPageError[0m[0;34m([0m[0mpageid[0m[0;34m=[0m[0mself[0m[0;34m.[0m[0mpageid[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;31mPageError[0m: Page id "ever3st" does not match any pages. Try another id!
PageError: Page id "ever3st" does not match any pages. Try another id!

